rest-client/src/main/java/com/gentics/mesh/rest/client/impl/MeshRestHttpClientImpl.java

```diff
@@ -143,6 +144,15 @@ public MeshRestClient disableAnonymousAccess() {
 		return prepareRequest(GET, "/" + encodeSegment(projectName) + "/nodes/" + uuid + getQuery(parameters), NodeResponse.class);
 	}
 
+	@Override
+	public MeshRequest<NodeResponse> upsertNode(String projectName, String uuid, NodeUpsertRequest nodeUpsertRequest,
```

Comment:
I know that the signature is different from `createNode(uuid, projectName)` but I prefer this order since it also reflects the ids in the REST path